docs/dg/projectManagement.md

```diff
@@ -21,6 +21,27 @@ This page contains information about project management tasks. The target audien
 
 <!-- ==================================================================================================== -->
 
+## Making a release on Github
+
+Before making a release, please check the following prerequisite.
+
+* Ensure that you have a proper version of **JDK `1.8.0`** installed (==Not **JDK `11`** or other later version==).
+* Ensure that the `JAVA_HOME` environment variable is correctly set to your JDK installation directory. You can refer to the [JDK Installation Guide](https://docs.oracle.com/cd/E19182-01/821-0917/inst_jdk_javahome_t/index.html).
+
+To make a release for RepoSense on Github, please follow the `Creating a release` section in the [Github Docs](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository).<br>
+  
+Take note of the following the section above:
+* When entering a release version number, use semantic versioning with some small tweak:
+  * Use `MAJOR.MINOR` as the version number when the release includes new features and/or major changes.
+  * Use `MAJOR.MINOR.PATCH` as the version number when the release only includes bug fixes and/or minor changes.
+  * Append `rc` to the version number to indicate that the release is a pre-release with no guarantee of stability.
```

Comment:
"with no guarantee of stability" I think should phrase it better with "that should not be used in production".

Reply:
Actually, I was thinking of directing the developer to another link to check the meaning of `rc` here. I will change it accordingly.